bin/retry-publish

```diff
@@ -76,13 +75,13 @@ Examples:
     server = CouchServer(config.AsyncTransfer.couch_instance,config.AsyncTransfer.opsProxy,config.AsyncTransfer.opsProxy)
     try:
         database = server.connectDatabase(config.AsyncTransfer.files_database)
-    except Exception, ex:
+    except Exception as ex:
         msg = "Error while connecting to %s couchDB." % config.AsyncTransfer.files_database
         print msg
         sys.exit(4)
     try:
-        docs_pubFailed = database.loadView('DBSPublisher','PublicationFailedByWorkflow',{'reduce':False,'key':taskname})['rows']
-    except Exception, ex:
+        docs_pubFailed = database.loadView('DBSPublisher','PublicationFailedByWorkflow',{'reduce':False,'startkey':[taskname,'']})['rows']
```

Comment:
It does not seem to work. You need to use the endkey as well: 

{'reduce':False,'startkey':[task], 'endkey':[task, {}]}